app/(tabs)/files.tsx

```diff
@@ -45,8 +45,23 @@ const FilesTabScreen = () => {
                 setLatestFiles(results);
             }
         };
+        const deleteFileTest = async () => {
+            const result = await deleteItem("/remote.php/dav/files/testuser/Documents/Readme.md");
+            console.log(result);
+        };
+        const createFolderTest = async () => {
+            const result = await createFolder("/remote.php/dav/files/testuser/Documents/TestFolder");
+            console.log(result);
+        };
+        const uploadFileTest = async () => {
+            const result = await uploadFile(new Blob(["Whoa, i'm such a great new blob"]), "/remote.php/dav/files/testuser/Documents/TestFolder/TestFile.txt");
+            console.log(result);
+        };
         fetchFolderContent();
         fetchLatestFiles();
+        deleteFileTest();
+        createFolderTest();
+        uploadFileTest();
```

Comment:
Thanks for putting this in here but better let's not merge this test dummy code into main. Do you agree or have you a more specific reason why we need this here?

Reply:
Should be fixed now, for quick reference one could always check out this comment anyways ;)